src/extension.ts

```diff
@@ -109,4 +114,95 @@ export const activate = () => {
         if (!getExtensionSetting('runOnSave') || reason === vscode.TextDocumentSaveReason.AfterDelay) return
         waitUntil(performFixes())
     })
+
+    vscode.workspace.onDidChangeTextDocument(
```

Comment:
The feature became larger, let's extrat it to sibling file like `commaOnEnter.ts` (and just do `registerCommaOnEnter()` in extension.ts), also don't forget to run `pnpx prettier --write src/**`